Exercises/removeDuplicates.js

```diff
@@ -0,0 +1 @@
+const removeDuplicates = (array) => [...new Set(array)];
```

Comment:
Using a Set is an efficient way to remove duplicates, as Sets automatically enforce uniqueness. Looks good.